### PR TITLE
Tachyons Bootstrap Link issue#45

### PR DESCRIPTION
**Pull request by nelsonic**
So much _fine_ work was put into Tachyons Bootstrap by these fine folks https://github.com/dwyl/tachyons-bootstrap/graphs/contributors ... ⏳ 
Feels like a _waste_ to not share it with the _thousands_ of people who are learning about tachyons through this tutorial...!
+ [x] Adds link to https://tachyons-bootstrap.dwyl.com #45 
+ [x] Adds link to Tachyons Cheatsheet #48 

This is what I do in my pomodoro _breaks_ ... 😉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +168,54 @@ What a _monolith_! There are a _lot_ of CSS attributes there; <br />
...
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  color: #800080; /* wikipedia.org/wiki/The_Color_Purple */
+}
+```
+3. Add the CSS class to the instance of the button you created.
+```html
+<button type="button" class="btn btn-primary custom-btn-purple-helvetica">
+  Primary</button>
+```
+This 3-step process is slow and _always_ ends up with
```
Comment (iteles) [inline README.md]: Missing the word 'more' here

File: README.md [hunk 2]
```diff
@@ -158,6 +168,54 @@ What a _monolith_! There are a _lot_ of CSS attributes there; <br />
...
+most high profile websites/apps have _hideous_ CSS Stats scores.
+This is just indicative of the wider problem.
+_All_ of Tachyons (_which is a complete framework_) is `72kb`
+(_with **everything**_) ...
+Medium is a content publishing company
+whose _one job_ is to present content effectively,
+and has hired some of the most talented designers and spent
+[$132M](https://www.crunchbase.com/organization/medium#section-overview)
+building the platform.
+If Medium can't get their CSS under control, who can?!
```
Comment (iteles) [inline README.md]: This adds quite a lot of pre-amble which could go in an appendix if it ends up being too much scrolling (with all the images and code snippets).

File: README.md [hunk 3]
```diff
@@ -430,6 +488,12 @@ Getting started with Tachyons is as easy as 1, 2, 3!
 We've created some examples for you to play around with
 before you decide to use it on your own projects.
 
+> If you want to see a quick-reference guide to Tachyons with detailed examples,
+checkout our Clone of Bootstrap in Tachyons:
+***Tachyons Bootstrap***: https://tachyons-bootstrap.dwyl.com
+
+If you want to learn hands-on follow this
```
Comment (iteles) [inline README.md]: Needs a little punctuation here `:`

File: README.md [hunk 4]
```diff
@@ -715,6 +779,8 @@ what the ideal dimensions are for the image they should upload.
 + [Tachyons Github Repo](https://github.com/tachyons-css/tachyons/) : https://github.com/tachyons-css/tachyons/
 + [Tachyons Verbose Version](https://github.com/tachyons-css/tachyons-display-verbose) : https://github.com/tachyons-css/tachyons-display-verbose
 + [Setting up Custom Tachyons Build](https://vimeo.com/174698456)
++ Tachyons Bootstrap Examples: https://#
```
Comment (iteles) [inline README.md]: Missing the URL here, should this be http://tachyons-bootstrap.dwyl.com/ ?

**Review by iteles — CHANGES_REQUESTED** [review]
@nelsonic I'll make these typo changes and push, no need for you to waste time on them.

File: README.md (hunk 2 above)
Comment (nelsonic) [reply README.md]: Removing it and adding issue https://github.com/dwyl/learn-tachyons/issues/51 to the backlog.

**Comment by nelsonic** [comment]
@iteles I think I've addressed all your comments. 👍

**Review by iteles — APPROVED** [review]
:+1: Thank you!

**Pull request merged**